handlers/workflow_handler_test.go

```diff
@@ -250,3 +251,26 @@ func TestShouldThrowErrorIfQueryParamsAreNotValidValuesInGetAllWorkflows(t *test
 	assert.NotNil(t, jsonResp)
 	assert.Equal(t, "page number or page size is not in proper format", jsonResp.Message)
 }
+
+func TestShouldThrowErrorIfFiltersAreNotInTheRightFormat(t *testing.T) {
```

Comment:
This is an integration test. These are expensive to run and manage. Here you have only covered negative path. If anything you should have a happy path at least.

Reply:
Alternatively, extracting error handling code into a common method and writing tests only once for that is better. All positive and negative tests can be covered in actual implementations' unit test